Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM alpine:3.1
+FROM alpine:edge
 MAINTAINER Miek Gieben <miek@miek.nl> (@miekg)
 
 RUN apk --update add bind-tools && rm -rf /var/cache/apk/*
+RUN apk --update add git mercurial go build-base && rm -rf /var/cache/apk/*
+RUN mkdir /go
+RUN export GOPATH="/go" && go get github.com/skynetservices/skydns && cd $GOPATH/src/github.com/skynetservices/skydns && go build -v
+RUN mv /go/bin/skydns /bin
+RUN rm -rvf /go
```

Comment:
Shouldn't all these lines be in a single `RUN`? Otherwise removing the files won't actually make the image smaller (they still end up in intermediate layers)